### PR TITLE
Mention how to add margins to Button's icon

### LINKED ISSUE #23828: Add a way to define margins for the icon in Button
There is no way to define margins for the icon in the Button class (as far a I know).

The buttons look visually awful without space:
image

I could offer myself to implement a proper solution for this but I'd like to hear some feedback about this. If there is a way but it is undocumented I could add it to the docs.

It would be good to know if there are more classes with this problem.

After searching I've found [this issue](https://github.com/godotengine/godot/issues/2772) but it's pretty old and I'm not sure it is a good solution.

### PR DESCRIPTION
**Pull request by KoBeWi**
Closes #23828

### CODE REVIEW

File: doc/classes/Button.xml [hunk 1]
```diff
@@ -78,6 +78,7 @@
 		</member>
 		<member name="icon" type="Texture2D" setter="set_button_icon" getter="get_button_icon">
 			Button's icon, if text is present the icon will be placed before the text.
+			To edit margin and spacing of the icon, use [code]hseparation[/code] theme property of [Button] and [code]content_margin_*[/code] properties of the used [StyleBox]es.
```
Comment (akien-mga) [inline doc/classes/Button.xml:81]: I guess we still lack support for `[theme_item hseparation]` even after #29868 ?

Comment (KoBeWi) [reply doc/classes/Button.xml:81]: Yeah.

Comment (pycbouh) [reply doc/classes/Button.xml:81]: I have a suspicion that operators and constructors don't have their own tags either :eyes:  (And they are also missing from doc status).

**Review by mhilbrunner — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.5.

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.1.

**Pull request merged**